contracts/src/L1/gateways/L1GatewayRouter.sol

```diff
@@ -140,6 +163,14 @@ contract L1GatewayRouter is OwnableUpgradeable, IL1GatewayRouter {
         return _amount;
     }
 
+    /// @inheritdoc IMessageDropCallback
+    function onDropMessage(bytes calldata _message) external payable virtual onlyInDropContext {
```

Comment:
Any concern for reentrancy here? (I assume no since `dropMessage` has `notInExecution`.)